diff --git a/third_party/boringssl/boringssl.gyp b/third_party/boringssl/borings
index 311fee4..70a80bc 100644
--- a/third_party/boringssl/boringssl.gyp
+++ b/third_party/boringssl/boringssl.gyp
@@ -60,9 +60,10 @@
       'defines': [
         'BORINGSSL_IMPLEMENTATION',
         'BORINGSSL_NO_STATIC_INITIALIZER',
+        'OPENSSL_NO_ASM',
         'OPENSSL_SMALL',
       ],
-      'dependencies': [ 'boringssl_asm' ],
+      #'dependencies': [ 'boringssl_asm' ],
       # TODO(davidben): Fix size_t truncations in BoringSSL.
       # https://crbug.com/429039
       'msvs_disabled_warnings': [ 4267, ],
diff --git a/third_party/boringssl/boringssl.gypi b/third_party/boringssl/boring
index 4efb2c1..4024eaf 100644
--- a/third_party/boringssl/boringssl.gypi
+++ b/third_party/boringssl/boringssl.gypi
@@ -314,6 +314,13 @@
       'src/crypto/x509v3/v3_skey.c',
       'src/crypto/x509v3/v3_sxnet.c',
       'src/crypto/x509v3/v3_utl.c',
+      'src/decrepit/bio/base64_bio.c',
+      'src/decrepit/blowfish/blowfish.c',
+      'src/decrepit/cast/cast.c',
+      'src/decrepit/cast/cast_tables.c',
+      'src/decrepit/des/cfb64ede.c',
+      'src/decrepit/rsa/rsa_decrepit.c',
+      'src/decrepit/xts/xts.c',
     ],
     'boringssl_linux_aarch64_sources': [
       'linux-aarch64/crypto/aes/aesv8-armx64.S',
