diff --git a/third_party/boringssl/boringssl.gypi b/third_party/boringssl/boringssl.gypi
index 7ae8461..b48de2c 100644
--- a/third_party/boringssl/boringssl.gypi
+++ b/third_party/boringssl/boringssl.gypi
@@ -304,6 +304,13 @@
       'src/crypto/x509v3/v3_skey.c',
       'src/crypto/x509v3/v3_sxnet.c',
       'src/crypto/x509v3/v3_utl.c',
+      'src/decrepit/bio/base64_bio.c',
+      'src/decrepit/blowfish/blowfish.c',
+      'src/decrepit/cast/cast.c',
+      'src/decrepit/cast/cast_tables.c',
+      'src/decrepit/des/cfb64ede.c',
+      'src/decrepit/rsa/rsa_decrepit.c',
+      'src/decrepit/xts/xts.c',
     ],
     'boringssl_linux_aarch64_sources': [
       'linux-aarch64/crypto/aes/aesv8-armx64.S',
